contracts/axelar-gateway/src/contract.rs

```diff
@@ -169,20 +186,20 @@ impl AxelarGatewayInterface for AxelarGateway {
         }
     }
 
+    // TODO: add docstring about how enforce_rotation_delay supposed to be used.
     fn rotate_signers(
         env: Env,
-        signers: axelar_soroban_interfaces::types::WeightedSigners,
-        proof: axelar_soroban_interfaces::types::Proof,
+        signers: WeightedSigners,
+        proof: Proof,
         enforce_rotation_delay: bool,
```

Comment:
```suggestion
        skip_rotation_delay: bool,
```
or `bypass_rotation_delay` or something similar that's opposite of enforcing the delay (which should be the default behaviour). While the implementation changed compared to the reference, a lack of name change here seems to be causing confusion